misc/fileTest.php

```diff
@@ -25,9 +25,7 @@
  * `php file-test.php /tmp/source-useragent.txt "not" "useragent" > /tmp/useragent-not-detected.txt`
  */
 use DeviceDetector\DeviceDetector;
-use DeviceDetector\Parser\Client\Browser;
 use DeviceDetector\Parser\Device\AbstractDeviceParser;
-use DeviceDetector\Parser\OperatingSystem;
 
 require __DIR__ . '/../vendor/autoload.php';
```

Comment:
I'm not sure that ' use ' will work if autoload is enabled just below.

Reply:
all is well, spl_autoload is started when the class is called, use ignores

Reply:
That works fine, as it will try to really use it the first time the class is used in the code...
It even doesn't fail if you define a class in a use statement that doesn't exist as long as you don't use it anywhere 🙈